randommenu_platinum.rb

```diff
@@ -0,0 +1,66 @@
+# Random Menu Generator
+
+# create empty arrays
+adjective = []
+cook_method = []
+food_item = []
```

Comment:
It might be good to have these variable names be plural, to indicate that there may be more than one item in each.